recipes-samples/images/lmp-feature-sysctl-net-queue-pfifo-fast.inc

```diff
@@ -0,0 +1,4 @@
+# packages
+CORE_IMAGE_BASE_INSTALL += " \
+    sysctl-net-queue-pfifo-fast \
+"
```

Comment:
We could rename this include to something like:
`lmp-feature-sysctl-set-default-packet-scheduler.inc`

With a default in the package of something like this:
`LMP_DEFAULT_PACKET_SCHEDULER ?= "fq_codel"`

For the gateway image the include would look like this:
```
...
require lmp-feature-sysctl-hang-crash-helper.inc

LMP_DEFAULT_PACKET_SCHEDULER = "pfifo_fast"
require lmp-feature-sysctl-set-default-packet-scheduler.inc
```